/trajectory_control_server_node:
  ros__parameters:
    add_acce_deccl_limits: true
    angular_gain: 0.5
    controller_rate: 10.0
    goal_reach_tol: 0.01
    linear_gain: 0.5
    max_angular_velocity: 2.0
    max_linear_acceleration: 0.1
    max_linear_deceleration: 0.1
    max_linear_velocity: 1.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false
    vertical_gain: 0.5
